readme.rst

```diff
@@ -66,13 +66,19 @@ From there the goal is to fix outstanding known issues and clean up the code.
 +----------+---------------------------+----------------------------+--------------+
 | Version  | hbmqtt compatibility      | Supported Python Versions  | PyPi Release |
 +----------+---------------------------+----------------------------+--------------+
-| 0.10.x   | YES - Drop-in Replacement | 3.7 - 3.9                  | 0.10.1       |
+| 0.10.x   | YES - Drop-in Replacement | 3.7*                       | 0.10.1       |
 +----------+---------------------------+----------------------------+--------------+
-| 0.11.x   | NO - Module renamed       | 3.7 - 3.10                 | No release   |
+| 0.11.x   | NO - Module renamed       | 3.8 - 3.10                 | No release   |
```

Comment:
`3.7` is still compatible with our code, but long term it would be nice if we could slowly phase out some older python versions

```suggestion
| 0.11.x   | NO - Module renamed       | 3.7 - 3.10                 | No release   |
```